backend/src/main/java/org/opensource/schulaltas/controller/model/PageDto.java

```diff
@@ -0,0 +1,24 @@
+package org.opensource.schulaltas.controller.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.opensource.schulaltas.model.component.Component;
+import org.springframework.data.annotation.Id;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class PageDto {
+
+ @Id
+ private String name;
+ private String author;
+ 
+ @JsonProperty ("components")
+ List<Component> componentList;
```

Comment:
```suggestion
private List<Component> components;
```
private and why not name it components. It is not a good idea to add the data structure to the name. when you change the data structure it could be very confusing

Reply:
Point! Will change that.